docs/src/pages/landing-page/Index.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <q-page class="bg-lp-dark text-white relative-position">
+
+    <section class="q-px-lg">
```

Comment:
Use semantic tags when you actually follow the standard reccomandation 👀 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section

Eg: each `<section>` is supposed to have an heading, which isn't exactly the case here
IMO it's better to just use divs in these cases

Reply:
I'm also unsure if these icons should be fixed to the page as the toolbar or scroll away, ask Nick about this (eg. via a comment on Zeplin)